assets/js/admin/form-builder.js

```diff
@@ -222,6 +223,290 @@
+
+			// Default value option change handler.
+			$( document.body ).on( 'input', '.everest-forms-field-option .everest-forms-field-option-row-default_value input', EVFPanelBuilder.updateRangeSliderBasicOptions );
+
+			// Use Text Prefix/Postfix option change handler (checkbox).
+			$( document.body ).on( 'change', '.everest-forms-field-option .evf-use-text-prefix-postfix', EVFPanelBuilder.updateRangeSliderBasicOptions );
+
+			// Text Prefix/Postfix option change handler (input).
+			$( document.body ).on( 'input', '.everest-forms-field-option .evf-input-prefix-text, .everest-forms-field-option .evf-input-postfix-text', EVFPanelBuilder.updateRangeSliderBasicOptions );
+
+			// Slider input visibility option change handler.
+			$( document.body ).on( 'change', '.everest-forms-field-option .evf-show-slider-input', EVFPanelBuilder.updateRangeSliderBasicOptions );
+
+			// Initialize Range Slider Fields.
+			$( '.everest-forms-field.everest-forms-field-range-slider' ).each( function( e ) {
+				var field_id = $( this ).data( 'field-id' );
+				EVFPanelBuilder.initializeRangeSliderField( field_id );
+			});
+		},
+
+		/**
+		 * Initialize a Range Slider Field.
+		 *
+		 * @since 1.7.0
+		 */
+		initializeRangeSliderField: function( field_id ) {
+			var $field = $( '#everest-forms-field-' + field_id );
+
+			// Initialize the field as an IonRangeSlider field.
+			$field.find( '.evf-range-slider-preview' ).ionRangeSlider();
+
+			// Slider handle/highlight/track color change handler.
+			EVFPanelBuilder.initializeSliderHandleColorOption( field_id );
+			EVFPanelBuilder.initializeSliderHighlightColorOption( field_id );
+			EVFPanelBuilder.initializeSliderTrackColorOption( field_id );
+			EVFPanelBuilder.updateRangeSliderBasicOptions( field_id );
+			EVFPanelBuilder.updateRangeSliderColors( field_id );
+		},
+
+		/**
+		 * Initialize Range Slider Handle Color Picker.
+		 *
+		 * @since 1.7.0
+		 */
+		initializeSliderHandleColorOption: function( field_id ) {
+			var $field_options_container = $( '#everest-forms-field-option-' + field_id );
+
+			// Initialize color picker for Handle Color option.
+			$field_options_container.find( '.evf-range-slider-handle-color' )
+			.wpColorPicker({
+				change: function( event, ui ) {
+					var new_color = $( event.target ).val();
+					var field_id = $( this ).closest( '.everest-forms-field-option-row' ).data( 'field-id' );
+					var current_skin = $field_options_container.find( '.evf-range-slider-skin' ).val();
+
+					EVFPanelBuilder.setSliderHandleColor( field_id, new_color, current_skin );
+				}
+			});
+		},
+
+		/**
+		 * Initialize Range Slider Highlight Color Picker.
+		 *
+		 * @since 1.7.0
+		 */
+		initializeSliderHighlightColorOption: function( field_id ) {
+			var $field = $( '#everest-forms-field-' + field_id );
+			var $field_options_container = $( '#everest-forms-field-option-' + field_id );
```

Comment:
Initialize these in a single statement.

Reply:
Resolved with 38e1bd4fdd984eab4133668204c60ca1ca725b73